.git-deps.yml

```diff
@@ -0,0 +1,128 @@
+deps:
```

Comment:
It would be great to rename this file to `ci.yml`, and `git-deps.py`  to `ci.py` so they clearly tell the user what they are used for. What do you think?